LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 brennerm@github
```

Comment:
@elsamuko Please change year to 2023 and name to _Max Brenner_

Reply:
Done